packages/wasm-terminal/src/command-runner/command-runner.ts

```diff
@@ -212,6 +218,10 @@ export default class CommandRunner {
     // Get our filesystem state
     const wasmFsJson = this.wasmTerminalConfig.wasmFs.toJSON();
 
+    // Create our Io Device Window
+    const sharedIoDeviceInputBuffer = new SharedArrayBuffer(8192);
```

Comment:
Why 8192?

Reply:
8192 is the length of the buffer. I'm giving it a bit of headroom so we don't run our of space in case we get a lot of input events at once 😄